pages/docs/configuration/configuration-export.md

```diff
@@ -173,9 +173,26 @@ pointDataSeries = loadParallelCSVSeries("A-ExporterTwo")
 
 ## Visualization with ParaView
 
+### Loading series files
+
+VTK, VTU, and VTP exporters write a `.series` file at the end of the simulation unless they are configured to export iterations.
+A series file loads the entire dataset and maps each export to its actual time.
+This allows you to open a series directly from the terminal, which can be convenient at times.
+
+{% version 3.3.0 %}
+New in version 3.3.0.
+{% endversion %}
```

Comment:
I am not sure what we have decided: Do we want to merge documentation PRs of upcoming features?

Does the `version` macro check if the version is unreleased?

Reply:
Yes, this is already covered: https://github.com/precice/precice.github.io/blob/55dcded2132af521142a1d15b74c41af910e6615/_plugins/customblocks.rb#L37